src/pages/Admin/Room/Edit/RoomEdit.tsx

```diff
@@ -134,24 +54,96 @@ const EditRoom: React.FC<EditRoomProps> = ({
 
       <Modal.Body>
         <Form onSubmit={handleSubmit(onSubmit)}>
-          {renderNameInput()}
-          {renderUrlInput()}
-          {renderDescriptionInput()}
-          {renderLogoInput()}
-
-          {roomTemplate?.customInputs?.map((input) => renderCustomInput(input))}
-
-          {renderTemplateSelect()}
+          <Form.Row>
+            <div className="input-wrapper">
+              <Form.Label>Name your room</Form.Label>
+              <Form.Control
+                type="text"
+                ref={register}
+                name="title"
+                placeholder="Room name"
+                custom
+              />
+            </div>
+          </Form.Row>
+
+          <Form.Row>
+            <div className="input-wrapper">
+              <Form.Label>The room url</Form.Label>
+              <Form.Control
+                type="text"
+                ref={register}
+                name="url"
+                placeholder="Room url"
+                custom
+              />
+            </div>
+          </Form.Row>
+
+          <Form.Row>
+            <div className="input-wrapper">
+              <Form.Label>The room description (optional)</Form.Label>
+              <Form.Control
+                type="text"
+                ref={register}
+                name="description"
+                placeholder="Description"
+                custom
+              />
+            </div>
+          </Form.Row>
```

Comment:
This block seems to differ from how it was previously defined in `renderDescriptionInput`. What was the reasoning for the differences?

Reply:
As an example, there was no `Form.Row` wrapping this before.

Reply:
At first, only the room name field used ```Form.Row``` and all the other blocks that are ```Form.Something```, so I added it to everything. The styling looked strange as different elements had a different amount of space around them. This seems to have fixed that (this probably should have been a new PR)

Before:
<img width="538" alt="Screen Shot 2021-02-22 at 9 33 06 PM" src="https://user-images.githubusercontent.com/51083763/108773044-b5efe580-7555-11eb-82b2-9c3531152b18.png">

After:
<img width="521" alt="Screen Shot 2021-02-22 at 9 32 38 PM" src="https://user-images.githubusercontent.com/51083763/108773091-c1dba780-7555-11eb-8f5b-64811518d035.png">

Reply:
This is the same reasoning I used when I added ```Form.Row``` etc to the all the other things that you commented.

Reply:
Cool, makes sense and sounds good. Thanks for explaining.

---

> this probably should have been a new PR

In future, that sort of thing probably would have been better as it's own standalone PR, 'focussed' on the intended changes.

As it is though, happy for you to leave it here now.